### PR TITLE
balance-phoenix0.1

### PR DESCRIPTION
**Pull request by warpdragon**
oaa levels only. a few nerfs and a buff. basic abilities do more damage to phoenix. supernova damages enemies more and tanks a few more hits.

### CODE REVIEW

**Review by DarkoniusXNG — CHANGES_REQUESTED** [review]
HP costs don't really need a nerf. Phoenix in general doesn't need a nerf. (I am sure Max will correct me if I am wrong) You may think that hp cost nerfs serve as a counter-weight for the ultimate buff but not really.

**Comment by TheUnholyWrath** [comment]
> HP costs don't really need a nerf. Phoenix in general doesn't need a nerf. (I am sure Max will correct me if I am wrong) You may think that hp cost nerfs serve as a counter-weight for the ultimate buff but not really.

HP costs scale inherently. Nerfing them just makes you not want to level up the ability. I don't think phoenix is strong enough that he deserves to be nerfed

File: game/scripts/npc/abilities/phoenix_supernova.txt [hunk 1]
```diff
@@ -59,13 +59,13 @@
...
         "var_type"                                        "FIELD_INTEGER"
-        "max_hero_attacks"                                "6 10 14 16 18" //OAA
+        "max_hero_attacks"                                "6 10 14 17 21" //OAA; Buff: OAA levels increased from 16 18; parallels max_hero_attacks_scepter
         "LinkedSpecialBonus"                              "special_bonus_unique_phoenix_1"
       }
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "max_hero_attacks_scepter"                        "7 12 17 19 21" //OAA
+        "max_hero_attacks_scepter"                        "7 12 17 20 24" //OAA; Buff: OAA levels increased from 19 21; parallels max_hero_attacks
```
[the remark this answers is not in the document]
Comment (warpdragon) [reply game/scripts/npc/abilities/phoenix_supernova.txt]: Okey doke.

File: game/scripts/npc/abilities/phoenix_supernova.txt [hunk 2]
```diff
@@ -48,7 +48,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_per_sec"                                  "60 120 180 360 540" //OAA
+        "damage_per_sec"                                  "60 120 180 400 650" //OAA; Buff: OAA levels increased from 360 540.
```
[the remark this answers is not in the document]
Comment (warpdragon) [reply game/scripts/npc/abilities/phoenix_supernova.txt]: Okey doke.

**Comment by warpdragon** [comment]
tested. doesn't crash game

**Comment by warpdragon** [comment]
this pr is all gummed up

**Pull request closed without merging**